ui-admin/src/study/participants/enrolleeView/EnrolleeOverview.tsx

```diff
@@ -71,29 +76,41 @@ export default function EnrolleeOverview({ enrollee, studyEnvContext, onUpdate }
     </InfoCard>
 
     {isLoadingRelations && <LoadingSpinner/>}
-    {
-      relations
-        .filter(relation => relation.relationshipType === 'PROXY')
-        .map(relation => {
-          return <InfoCard key={relation.id}>
-            <InfoCardHeader>
-              <InfoCardTitle title={'Proxy'}/>
-            </InfoCardHeader>
-            <InfoCardBody>
-              <InfoCardValue
-                title={'Name'}
-                values={
-                  [formatName(relation.enrollee?.profile)]
-                }
-              />
-              <InfoCardValue
-                title={'Contact Email'}
-                values={[relation.enrollee?.profile?.contactEmail || '']}
-              />
-            </InfoCardBody>
-          </InfoCard>
-        })}
-
+    { proxyRelations.length > 0 && <InfoCard>
+      <InfoCardHeader>
+        <InfoCardTitle title={'Proxies'}/>
+      </InfoCardHeader>
+      <InfoCardBody>
+        <ul>
+          {proxyRelations.map(relation => <li className="mt-2" key={relation.id}>
+            <span className="fw-bold me-3">{formatName(relation.enrollee!.profile) }</span>
+            <span className="me-3">{relation.enrollee?.profile?.contactEmail}</span>
+              (<Link to={studyEnvParticipantPath(paramsFromContext(studyEnvContext), relation.enrolleeId)}>
```

Comment:
more curious than feedback, why not link with the shortcode? It looks like it does an auto-refresh when you use an ID which causes a little jitter